dependencies/registry/certificate.yaml

```diff
@@ -0,0 +1,19 @@
+---
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: registry-cert
+  namespace: kind-registry
+spec:
+  isCA: true
```

Comment:
I don't think that this field is needed. It's needed when certificates can be generated out of this certificate, not when using it with an end service.

Reply:
I have a strong recollection that this field was needed last time we played with that. We were missing it and it took us a long time to figure out that this was the thing that was missing.
@avi-biton do you recall more details?

Reply:
I think it is needed for self signed certificates